crates/cosmos/cosmos-relayer/src/presets/chain.rs

```diff
@@ -1,20 +1,23 @@
 #[cgp::re_export_imports]
 mod preset {
-    use hermes_cosmos_chain_components::components::client::*;
-    use hermes_cosmos_chain_components::components::transaction::*;
-    use hermes_cosmos_test_components::chain::components::*;
+    use hermes_cosmos_chain_components::components::client::CosmosChainClientPreset;
+    use hermes_cosmos_chain_components::components::transaction::CosmosChainTxPreset;
+    use hermes_cosmos_test_components::chain::components::CosmmosChainTestPreset;
```

Comment:
```suggestion
    use hermes_cosmos_test_components::chain::components::CosmosChainTestPreset;
```

Reply:
Nice catch! Fixed in 93801ad07861b6c4c3aa0843a1a113e5c52fe63d.